runtime_scan/pkg/orchestrator/scanwatcher/helpers.go

```diff
@@ -28,9 +28,17 @@ func isScanTimedOut(scan *models.Scan, timeout time.Duration) bool {
 	if scan == nil || scan.StartTime == nil {
 		return false
 	}
-	deadline := (*scan.StartTime).Add(timeout)
 
-	return time.Now().UTC().After(deadline)
+	now := time.Now().UTC()
+
+	// Set timeoutTime to global timeout provided to the ScanWatcher controller
+	timeoutTime := scan.StartTime.Add(timeout)
```

Comment:
maybe it's better to do it in an else of the if, also rename the param timeout->globalTimeout following your comment?

Reply:
Honestly, I'd keep it as is since it is easy to understand even without the comments. I prefer to not use if-else until it is absolutely necessary.